lib/money/bank/variable_exchange.rb

```diff
@@ -216,14 +217,7 @@ def export_rates(format, file = nil, opts = {})
           RATE_FORMATS.include? format
 
         store.transaction do
-          s = case format
-          when :json
-            JSON.dump(rates)
-          when :ruby
-            Marshal.dump(rates)
-          when :yaml
-            YAML.dump(rates)
-          end
+          s = FORMAT_IMPORTERS[format].send(:dump, rates)
```

Comment:
Is the use of `send` here necessary? The `dump` method seemed to be public judging by the previous code, so doing `FORMAT_IMPORTERS[format].dump(rates)` should work.